services/apps/src/apps/routes/schemas/groups.clj

```diff
@@ -0,0 +1,72 @@
+(ns apps.routes.schemas.groups
+  (:use [common-swagger-api.schema :only [describe NonBlankString]])
+  (:require [schema.core :as s]))
+
+(s/defschema Group
```

Comment:
Thanks, yeah, that's probably a good idea. I'll create a Jira issue for it.

On Fri, Aug 12, 2016 at 12:09 PM Paul Sarando <notifications@github.com>
wrote:

> In services/apps/src/apps/routes/schemas/groups.clj
> <https://github.com/cyverse/DE/pull/309#discussion_r74645126>:
>
> > @@ -0,0 +1,72 @@
> > +(ns apps.routes.schemas.groups
> > +  (:use [common-swagger-api.schema :only [describe NonBlankString]])
> > +  (:require [schema.core :as s]))
> > +
> > +(s/defschema Group
>
> Some of the schemas in this namespace seem like duplicates of schemas in
> iplant-groups
> <https://github.com/cyverse/DE/tree/8fba9b30622796c94e8003a051e77422aef1b931/services/iplant-groups/src/iplant_groups/routes/schemas>
> .
> These common schemas could be moved into the common-swagger-api lib so
> that iplant-groups and apps can share them.
> For example, some ontology schemas
> <https://github.com/cyverse/DE/tree/8fba9b30622796c94e8003a051e77422aef1b931/libs/common-swagger-api/src/common_swagger_api/schema/ontologies.clj>
> are shared between the apps and metadata services.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/cyverse/DE/pull/309/files/dbd97aa26b0b84b27109df1ab6fd963a48f26f53#r74645126>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAZVpxQ5wnKWoAAVEF48nBD_WNZas3Fiks5qfMTsgaJpZM4JirYF>
> .
>